### ISSUE TITLE
Tailnet documentation - default vs fun names

### ISSUE THREAD

**Issue by j0sh** [tailscale/tailscale#7561]
### What is the issue?

The [tailnet documentation](https://tailscale.com/kb/1217/tailnet-name) is a little unclear on whether nodes will still be accessible using the default tailnet name after setting a fun name. In one place, the docs say this:

> Your default name always exists for your tailnet, even if you create a fun tailnet name.

This note implies a fun OR default tailnet name, not a fun AND default tailnet name:

> If you have a fun name, you can set your tailnet name to either the fun name or the default name in the [DNS page](https://tailscale.com/admin/dns) of the admin console.

While in another place, the docs say:

> When you create a fun tailnet name, it becomes your active tailnet name. If you previously used your default tailnet name for [MagicDNS](https://tailscale.com/kb/1081/magicdns/), [HTTPS certificates](https://tailscale.com/kb/1153/enabling-https/), or [sharing](https://tailscale.com/kb/1084/sharing/), existing links to devices in your tailnet may no longer work.

I'll also say that MagicDNS is enabled in our account _but_ I have no idea whether it's actually being used. Eg, I frequently visit `https://<node>.<default-tailnet>.ts.net` and things Just Work! That's a good thing, I suppose?

Will that URL keep working after setting a fun name? FWIW, I manually provision HTTPS certificates for that endpoint right now. What about other services that rely on the Tailscale node name (eg, DB connection strings)?

For context - what I would really like to do is to have an "internal" tailnet name (the default name is fine for this) and a "public" name to share more broadly with Tailscale Funnel. The default / fun tailnet names seem to be a nice way to make that distinction, but also open to other ways to do this.

### Steps to reproduce

1. Read the documentation at https://tailscale.com/kb/1217/tailnet-name
2. Be puzzled

**Comment by mayakacz** [comment]
> The tailnet documentation is a little unclear on whether nodes will still be accessible using the default tailnet name after setting a fun name. 
> Will that URL keep working after setting a fun name? 

No, it currently works such that it can only have one of the names used for things like MagicDNS.

> For context - what I would really like to do is to have an "internal" tailnet name (the default name is fine for this) and a "public" name to share more broadly with Tailscale Funnel. The default / fun tailnet names seem to be a nice way to make that distinction, but also open to other ways to do this.

Leaving this open as a feature request.
